feat: add `zetachain` metadata

Azleal opened the pull request:
add zetachain icon

DanielSinclair commented:
@Azleal Do you happen to have a ZetaChain SVG with an opaque colored background?

Azleal commented:
> @Azleal Do you happen to have a ZetaChain SVG with an opaque colored background?

@DanielSinclair I updated the zetachain svg logo with an opaque colored background. also added zetachain to site project.

magiziz reviewed (approved):
Looks good 👍

Did a small tweak and renamed testnet zetachain to `Zeta Athens` instead of `ZetaChain Athens` since it might cause overflow UI bug to the modal.
